source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
+}
+
+.figure__image {
```

Comment:
The styles inside the`.figure__image` selector should be moved to be applied to all `image` pattern instances from within the `image` pattern's styles, as these should always apply to a content image, whether or not the image is in a figure. It also ends up making the twig code simpler as we then don't need to select for the image type of figure in order to apply these styles (figures may not be images, they may be videos, tables etc instead). We can always revisit if an edge-case arises.